docs/_deploying/heroku.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Deploying to Heroku
+order: 0
+slug: heroku
+lang: en
+layout: tutorial
+permalink: /deploying/heroku
+redirect_from:
+  - /deploying
+---
+# Deploying to Heroku
+
+<div class="section-content">
+This guide will walk you through preparing and deploying a Slack app using [Bolt for JavaScript](/bolt-js) and the [Heroku platform](https://heroku.com/). Along the way, we’ll download a Bolt Slack app, prepare it for Heroku, and deploy it.
+</div>
+
+When you’re finished, you’ll have this ⚡️[Getting Started with Heroku app](https://github.com/mwbrooks/bolt-js-getting-started-with-heroku-app) to run, modify, and make your own.
```

Comment:
@shaydewael while writing this guide, I was curious about your preference on using contractions like _you're_ and _you'll._ Do you try to avoid contractions or use them liberally?

Reply:
I'm fine with contractions! Sometimes I'll avoid them for clarity, but besides that I use them plenty.

Reply:
Great to know! This will be super helpful to me while I write the other deployment guides 🙇🏻